frontend/src/scenes/insights/ActionFilter/ActionFilterRow/ActionFilterRow.tsx

```diff
@@ -480,6 +481,7 @@ function MathSelector({
             onChange={(value) => onMathSelect(index, value)}
             data-attr={`math-selector-${index}`}
             dropdownMatchSelectWidth={false}
+            dropdownStyle={{ width: 320 }}
```

Comment:
This makes the dropdown considerably larger even if no upsell is being shown.

<img width="339" alt="" src="https://user-images.githubusercontent.com/5864173/144945318-6f045c8f-cdcf-4b8b-b1cd-d19f1ed5c38c.png">

Reply:
It still looked OK to me w/o it and didn't want to add complexity here due to it. Disagree?

Reply:
I think it's a fairly easy change (if we're showing the upsell add the width thing, otherwise leave this undefined), and it does look better to have it properly adjust, but up to you.

Reply:
I didn't want to bring "if we're showing this or not" into the parent components if I could help it. maxWidth works though!